lang/test/org/partiql/lang/eval/evaluatortestframework/AstEvaluatorTestAdapterTests.kt

```diff
@@ -0,0 +1,417 @@
+                    expectedPermissiveModeResult = "\$partiql_missing::null",
+                    expectedResultFormat = ExpectedResultFormat.ION
+                ),
+                EvaluationSession.standard()
+            )
+        }
+    }
+
+    @Test
+    fun `runEvaluatorTestCase - expected result matches - ExpectedResultFormat-ION (missing)`() {
+        assertDoesNotThrow("happy path - should not throw") {
+            testAdapter.runEvaluatorTestCase(
+                EvaluatorTestCase(
+                    query = "MISSING",
+                    expectedResult = "\$partiql_missing::null",
+                    expectedResultFormat = ExpectedResultFormat.ION
+                ),
+                EvaluationSession.standard()
+            )
+        }
+    }
+
+    @Test
+    fun `runEvaluatorTestCase - expected result matches - ExpectedResultFormat-ION (date)`() {
+        assertDoesNotThrow("happy path - should not throw") {
+            testAdapter.runEvaluatorTestCase(
+                EvaluatorTestCase(
+                    query = "DATE '2001-01-01'",
+                    expectedResult = "\$partiql_date::2001-01-01",
+                    expectedResultFormat = ExpectedResultFormat.ION
+                ),
+                EvaluationSession.standard()
+            )
+        }
+    }
+
+    @Test
+    fun `runEvaluatorTestCase - expected result matches - ExpectedResultFormat-ION (time)`() {
+        assertDoesNotThrow("happy path - should not throw") {
+            testAdapter.runEvaluatorTestCase(
+                EvaluatorTestCase(
+                    query = "TIME '12:12:01'",
+                    expectedResult = "\$partiql_time::{hour:12,minute:12,second:1.,timezone_hour:null.int,timezone_minute:null.int}",
+                    expectedResultFormat = ExpectedResultFormat.ION
+                ),
+                EvaluationSession.standard()
+            )
+        }
+    }
+
+    @Test
+    fun `runEvaluatorTestCase - expected result matches - ExpectedResultFormat-ION_WITHOUT_BAG_AND_MISSING_ANNOTATIONS mode (int)`() {
+        assertDoesNotThrow("happy path - should not throw") {
+            testAdapter.runEvaluatorTestCase(
+                EvaluatorTestCase(
+                    query = "1",
+                    expectedResult = "1",
+                    expectedResultFormat = ExpectedResultFormat.ION_WITHOUT_BAG_AND_MISSING_ANNOTATIONS
+                ),
+                EvaluationSession.standard()
+            )
+        }
+    }
+
+    @Test
+    fun `runEvaluatorTestCase - different permissive mode result - ExpectedResultFormat-ION_WITHOUT_BAG_AND_MISSING_ANNOTATIONS`() {
+        assertDoesNotThrow("happy path - should not throw") {
+            testAdapter.runEvaluatorTestCase(
+                EvaluatorTestCase(
+                    query = "1 + MISSING",
+                    expectedResult = "null",
+                    // note: In this ExpectedResultFormat we lose the f that this MISSING and not an
```

Comment:
typo: "... lose the **fact** that this **is** MISSING ..."

```suggestion
                    // note: In this ExpectedResultFormat we lose the fact that this is MISSING and not an
```